uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/QuotesRecyclerView.kt

```diff
@@ -70,12 +70,38 @@ class QuotesRecyclerView @JvmOverloads constructor(context: Context, attr: Attri
     }
 
     override fun setListVisibility(visible: Boolean) {
-        quotesListRecycler.visibility = if (visible) View.VISIBLE else View.GONE
-        noDestinationLabel.visibility = if (visible) View.GONE else View.VISIBLE
-        noDestinationVehiclesLabel?.visibility = if (visible) View.GONE else View.VISIBLE
+        if (visible) {
+            quotesListRecycler.visibility = View.VISIBLE
+            noDestinationLabel.visibility = View.GONE
```

Comment:
The single-line `visibility` assignment is better (as done in the old code). It's easier to understand the logic. You have to read 1 line instead of 2. When you read 2 lines, you have to be careful if they are the same labels in both if branches. I think it should be a good convention to use single liners, when possible.

Please check the other instances where this not done one a single line.

Reply:
Not sure if checking on each line the same condition is a good practice since each time a new line is reached, the check is repeatedly verified. 
What do you think @cosminBadulescu @mohadian ?